enterprise/server/registry/registry.go

```diff
@@ -413,8 +422,8 @@ func (r *registry) writeManifest(ctx context.Context, oldManifest partial.Descri
 	return nil
 }
 
-func (r *registry) convert(ctx context.Context, req *request, remoteDesc *remote.Descriptor) (*regpb.Manifest, error) {
-	req.logger.Infof("Converting %q", remoteDesc.Ref)
+func (r *registry) convert(ctx context.Context, reqLogger log.Logger, remoteDesc *remote.Descriptor) (*regpb.Manifest, error) {
```

Comment:
I'd prefer it if we didn't pass loggers around -- it's more allocations and less readability for not a ton of gain. I understand the need to sometimes have a custom logger that logs additional stuff, but the idea of doing it per-request is too much.

But if you wanted to log request ID in the logger (like I think you're doing here?), I'd def be into variants of log.Printf/log.Warningf/etc that take a context and add a UUID tag?